### ISSUE TITLE
CORS Issue with Solaire Wallet Integration

### ISSUE THREAD

**Issue by lucaspan** [authenticvision/digitalsoul#78]
Token metadata and token image URL CORS is not enabled. Therefore, loading NFT images in Solaire web wallet will be blocked by browsers (e.g Chrome). Please see CORS test below:

https://cors-test.codehappy.dev/?url=https%3A%2F%2Fquintinius.xyz%2Fapi%2Fv1%2Fcollections%2FMADSSB030%2F0x1d73d67eafde647d8f006ebf81c6fa170663d084f5c44a81d841ab66d6230709&origin=https%3A%2F%2Fwallet.sandbox.mysolaire.xyz%2F&method=get

https://cors-test.codehappy.dev/?url=https%3A%2F%2Fquintinius.xyz%2Fapi%2Fv1%2Fassets%2FMADSSB030%2F0x574585635d3e654f42dc0529c8c91d3dd5cd7501b22c3b6a27e3fef71f31961c&origin=https%3A%2F%2Fwallet.sandbox.mysolaire.xyz%2F&method=get

**Comment by tbergmueller** [comment]
- Implemented https://github.com/authenticvision/digitalsoul/pull/79 
- Deployed to testing-infrastructure (links above)
- CORS-Test is now happy, see screenshot

![image](https://github.com/authenticvision/digitalsoul/assets/2257335/47b5595b-8d6a-4638-a25d-3832c4169da9)


@lucaspan if you can confirm this works with Solaire-Wallets now as intended, we'll proceed to merge into master and deploy to production.

**Comment by tbergmueller** [comment]
Has been confirmed by Solaire to work
